sp800-63-3/sec5_DIRM.md

```diff
@@ -44,31 +44,31 @@ A summary of each of the identity, authenticator, and federation assurance level
 |:----------------------|
 | **IAL1** - At IAL1, attributes, if any, are self-asserted or should be treated as self-asserted.|
 | **IAL2** - At IAL2, either remote or in-person identity proofing is required. IAL2 requires identifying attributes to have been verified in person or remotely using, at a minimum, the procedures given in [[SP 800-63A]](sp800-63a.html).|
-| **IAL3** - At IAL3, in-person identity proofing is required. Identifying attributes must be verified by an authorized representative of the CSP through examination of physical documentation as described in [[SP 800-63A]](sp800-63a.html).|
+| **IAL3** - At IAL3, in-person identity proofing is required. Identifying attributes must be verified by an authorized CSP representative through examination of physical documentation as described in [[SP 800-63A]](sp800-63a.html).|
```

Comment:
We had taken the printed brackets off inline uses of document numbers, probably should here too.